pkg/image/graph/nodes/types.go

```diff
@@ -135,6 +139,10 @@ func (n DockerImageRepositoryNode) String() string {
 	return string(DockerImageRepositoryNodeName(n.Ref))
 }
 
+func (n DockerImageRepositoryNode) ResourceString() string {
+	return "dockerrepository/" + n.String()
```

Comment:
it is possible that the only user of that method / type will get removed based on an exchange with @bparees in bc.go; short term i'll replace with the docker pull spec